discord/cog.py

```diff
@@ -739,16 +742,41 @@ def _resolve_name(self, name: str, package: Optional[str]) -> str:
         except ImportError:
             raise errors.ExtensionNotFound(name)
 
-    def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
-        """Loads an extension.
+    def load_extensions_from_path(
+        self, *paths: Union[str, pathlib.Path], recursive: bool = False, must_exist: bool = True
+    ) -> None:
```

Comment:
I feel like this should return a list of all of the extensions loaded just in case the bot developer wants to unload or reload all of them.